### ISSUE TITLE
Format of public.openTypeMeta

### ISSUE THREAD

**Issue by schriftgestalt** [unified-font-object/ufo-spec#187]
I think [public.openTypeMeta](https://unifiedfontobject.org/versions/ufo3/lib.plist/#publicopentypemeta) should contain a list instead a dict. See: https://github.com/fonttools/fonttools/issues/2345

**Comment by benkiel** [comment]
@moyogo this was your part of the spec: are you ok with this being a `list` and not a `dict`?

**Comment by benkiel** [comment]
(also, would this would be a breaking change, so we'd need to put it in UFO4, no?)

**Comment by moyogo** [comment]
@benkiel yes, @schriftgestalt is correct. It should have been a <s>dict</s> [Edit: list].

**Comment by schriftgestalt** [comment]
(Just to be clear, it should be a list, not a dict.)

**Comment by benkiel** [comment]
@moyogo @anthrotype Looking back at this, because it's a public key, I don't think we need to do it in UFO4, but this would be breaking to go to `list`. Would you be ok with changing the definition here now, or would you rather wait for a UFO number bump?

**Comment by anthrotype** [comment]
I'd vote to wait for version bump since I haven't seen concrete use cases for the duplicate tags yet, and breaking existing projects would be worse

**Comment by anthrotype** [comment]
and fonttools meta implementation hasn't been fixed either...

**Comment by benkiel** [comment]
Got it. Cool.

**Comment by NightFurySL2001** [comment]
Actually, given that `dlng` and `slng` are internally strings, will it be reasonable to assume the type is also a string in addition to a list? Currently FontCreator exports `dlng` and `slng` directly as a comma-separated list, as typed by the user in the UI.

```xml
  <key>public.openTypeMeta</key>
  <dict>
    <key>dlng</key>
    <string>zh, Hant, zh-Hant, nan, hak, yue, nan-Latn, nan-Latn-pehoeji, nan-Latn-tailo, Bopo, Hanb</string>
    <key>slng</key>
    <string>zh, Hans, Hant, zh-Hans, zh-Hant, nan, hak, yue, Bopo, Hanb, zh-Latn, zh-Latn-pinyin, nan-Latn, nan-Latn-pehoeji, nan-Latn-tailo, Latn, Cyrl, Grek</string>
  </dict>
```
